src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Webpack</title>
+    <!-- <link rel="stylesheet" href="style.css"> -->
```

Comment:
- [ OPTIONAL] Kindly make sure to delete commented out lines of code to make your project clean :recycle: